examples/notebooks/parameter-management.ipynb

```diff
@@ -0,0 +1,383 @@
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# In directory /tmp\n",
+    "import pybamm\n",
+    "pybamm.PARAMETER_PATH"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "which means that parameter files will first be searched in `/tmp` (the current directory) and the `input/parameters` directory inside the \n",
+    "PyBaMM installation directory."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Editing the default parameters\n",
+    "It is often much easier to define new parameters by editing exiting ones.\n",
+    "\n",
+    "We recommend not to alter the default parameters provided with the PyBaMM installation. Instead, you can pull all parameter set for\n",
+    "a given chemistry into the current directory:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Directory lithium-ion does not exist.\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash\n",
+    "if [[ ! -d \"lithium-ion\" ]]; then\n",
+    "echo \"Directory lithium-ion does not exist.\"\n",
+    "fi"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "pybamm_edit_parameter lithium-ion"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The above commands will create a `lithium-ion` directory in the current directory, populated with copies of the default parameters, for editing:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "anodes\n",
+      "cathodes\n",
+      "cells\n",
+      "electrolytes\n",
+      "experiments\n",
+      "seis\n",
+      "separators\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash\n",
+    "ls lithium-ion"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As an example, let's create a new parameter file for the `cell` component, for the `lithium-ion` chemistry:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "mkdir lithium-ion/cells/my_new_param_set"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create example parameter file\n",
+    "f = open(\"lithium-ion/cells/my_new_param_set/param_file.csv\", \"w+\")\n",
+    "f.write(\n",
+    "\"\"\"\n",
+    "Name [units],Value\n",
+    "a, 4\n",
+    "b, 5\n",
+    "c, 6\n",
+    "\"\"\"\n",
+    ")\n",
+    "f.close()"
```

Comment:
Maybe we should add a line saying that this can be done as well by editing an existing file or creating a new one using any editor (some users might be confused by this notation).